### PR TITLE
Create API for handling BigInts

### PR DESCRIPTION
**Pull request by zherczeg**
#4109 should land first

### CODE REVIEW

File: docs/02.API-REFERENCE.md [hunk 1]
```diff
@@ -3643,6 +3750,133 @@ main (void)
...
+  in runtime with the `JERRY_FEATURE_BIGINT` feature enum value,
+  see: [jerry_is_feature_enabled](#jerry_is_feature_enabled).
+- Returned value must be freed with [jerry_release_value](#jerry_release_value) when it
+  is no longer needed.
+
+**Prototype**
+
+```c
+void
+jerry_get_bigint_digits (jerry_value_t value, uint64_t *digits_p, uint32_t size, bool *sign)
```
Comment (dbatyai) [inline docs/02.API-REFERENCE.md:3838]: sign_p

File: jerry-core/api/jerry.c [hunk 2]
```diff
@@ -3405,6 +3484,60 @@ jerry_get_symbol_descriptive_string (const jerry_value_t symbol) /**< symbol val
...
+} /* jerry_get_bigint_size_in_digits */
+
+/**
+ * Get the uint64 digits of a BigInt value (lowest digit first)
+ */
+void
+jerry_get_bigint_digits (jerry_value_t value, /**< BigInt value */
+                         uint64_t *digits_p, /**< [out] buffer for digits */
+                         uint32_t size, /**< buffer size in digits */
+                         bool *sign) /**< [out] sign of BigInt */
```
Comment (dbatyai) [inline jerry-core/api/jerry.c:3517]: sign_p

File: jerry-core/ecma/operations/ecma-bigint.c [hunk 3]
```diff
@@ -397,34 +398,222 @@ ecma_bigint_number_to_bigint (ecma_number_t number) /**< ecma number */
...
+} /* ecma_bigint_get_size_in_digits */
+
+/**
+ * Get the uint64 digits of a BigInt value
+ */
+void
+ecma_bigint_get_digits (ecma_value_t value, /**< BigInt value */
+                        uint64_t *digits_p, /**< [out] buffer for digits */
+                        uint32_t size, /**< buffer size in digits */
+                        bool *sign) /**< [out] sign of BigInt */
```
Comment (dbatyai) [inline jerry-core/ecma/operations/ecma-bigint.c:560]: sign_p

**Comment by zherczeg** [comment]
Thnak you, patch is updated.

**Pull request merged**